### DISCUSSION TITLE
Feature Suggestion: Cancel My Transaction Button

### DISCUSSION THREAD

**Discussion by jonnygrove** [arnoudkooi/SN-Utils#335] [Ideas]
I have a suggestion for a new button to cancel any long-running transactions:
Essentially just a button that would redirect the current page (or open a new window) to the cancel transaction page: 
cancel_my_transaction.do

I've had to use this a few times when a large report or dashboard gets hung up. Having a quick button on the extension might help without having to manually put the url in.

Per https://support.servicenow.com/kb?id=kb_article_view&sysparm_article=KB0656014

**Comment by arnoudkooi** [comment]
There already is a slashcommand `/cancel` and available via the right-click context menu under tools.
You can 
Checkout the [cheatsheet (video)](https://www.arnoudkooi.com/cheatsheet/) to know more about slashcommands


See the screenshots below and let me know if it works!


Standard slashcommand
![Google Chrome20221209 - 1833](https://user-images.githubusercontent.com/840781/206759842-379e6e3a-7caf-4313-be1b-dc7097ef06e3.png)

Via contextmenu
![Google Chrome20221209 - 1837](https://user-images.githubusercontent.com/840781/206760557-b8f0e9bd-350f-48a2-83c2-cab76e3193a0.png)

Customize order to always have this one at top
![Google Chrome20221209 - 1854](https://user-images.githubusercontent.com/840781/206763475-21648f88-4bb0-4a83-b415-cbd074492d32.png)

**Comment by arnoudkooi** [comment]
Hope this worked, Ill mark it solved
